subprojects/android-test/test-report/src/main/kotlin/com/avito/android/test/report/screenshot/ScreenshotCapturerImpl.kt

```diff
@@ -35,17 +44,21 @@ class ScreenshotCapturerImpl(private val outputDirectory: Lazy<File>) : Screensh
         compressFormat: Bitmap.CompressFormat,
         quality: Int
     ): Result<File?> {
-        return captureBitmap().flatMap { capture ->
-            Result.Success(
-                when (capture) {
-                    is Capture.Bitmap -> File(outputDirectory.value, filename).also { file ->
-                        FileOutputStream(file).use {
-                            capture.value.compress(compressFormat, quality, it)
-                        }
+        return reportFileProvider.provideReportDir().flatMap { dir ->
```

Comment:
I will fail all screenshots because of dir could we guarantee dir existence on screenshot capturer creation?